src/python/pants/backend/terraform/lint/tfsec/tfsec.py

```diff
@@ -40,10 +41,21 @@ class TFSec(ExternalTool):
         default=None,
         advanced=True,
         help=lambda cls: softwrap(
-            f"""
+            """
             Path to the tfsec config file (https://aquasecurity.github.io/tfsec/latest/guides/configuration/config/)
 
-            Setting this option will disable `[{cls.options_scope}].config_discovery`. Use this option if the config is located in a non-standard location.
+            Setting this option will disable config discovery for the config file. Use this option if the config is located in a non-standard location.
+            """
+        ),
+    )
+    custom_check_dir = DirOption(
```

Comment:
I know nothing about tfsec or terraform, but might someone have multiple directories containing custom checks?

If they do, I could imagine:

- wanting to specify multiple here
- wanting to partition by those dirs, e.g. `app1/custom-checks` should apply to `app1`, and `app2/custom-checks` should apply to `app2`

If this is possible, potentially this could be something like `custom_check_dir_globs` or something like that? Or, taken to an extreme: have an tailored target for each custom checks dir, rather than configure it globally.

As I said, I know nothing about tfsec, so this could all be nonsense!

Thoughts?

Reply:
I would say that's typically not how people operate; the default is having a `.tfsec/` in your repo root, with the custom check file inside it. You could opt to have multiple dirs, each with their own `.tfsec`, and run `tfsec` from each of those dirs separately, but I haven't seen anyone do that. If requested, it could be an enhancement later on IMO.